instrumental/drivers/motion/apt.py

```diff
@@ -127,7 +128,7 @@ def move_completed(self):
         """ Check if the move is completed.
         """
         rsp = self._ser.read(6)
-        if rsp == bytes([0x64, 0x04, 0x0e, 0x00, self.src | 0x80, self.dst ]):
+        if rsp[:4] == bytes([0x64, 0x04, 0x0e, 0x00, self.src | 0x80, self.dst ])[:4]:
```

Comment:
Can you add a short comment here (and at the other location) to indicate why we're only taking the first 4 bytes? This is the sort of info that's easy to forget when we want to change the code in the future.